lib/ash_json_api/api/api.ex

```diff
@@ -25,6 +25,20 @@ defmodule AshJsonApi.Api do
         type: :string,
         doc: "The route prefix at which you are serving the JSON:API"
       ],
+      open_api: [
```

Comment:
So what we want to do here is add a "nested section". So that would look like:

```elixir
@open_api %Spark.Dsl.Section{
  name: :open_api,
  schema: [
    tag: ....
  ]
}
```

Then put it in the json_api section as `sections: [@open_api]`.

Then we'd use `Spark.Dsl.Extension.get_opt(...)` to get the option out of the `[:json_api, :open_api]` section path